panda_grasp:
  base_frame_id: panda_link0
  tool0_frame_id: panda_link8
  T_tool0_tcp: {rotation: [0.000, 0.000, -0.382, 0.924], translation: [0.000, 0.000, 0.065]}
  finger_depth: 0.05
  max_opening_width: 0.08
  cam:
    frame_id: camera_depth_optical_frame
    topic_name: /camera/depth/image_rect_raw
    intrinsic:
      height: 480
      width: 640
      K: [383.265, 0.0, 319.39, 0.0, 383.26, 242.43, 0.0, 0.0, 1.0]
